data/preset/news.yaml

```diff
@@ -42,7 +42,7 @@ defines:
     contents:
     - type: ContentGroup.DynamicTitle
       properties:
-        format-string: !translate "see more from %s"
+        format-string: !translate "More news from %s"
```

Comment:
That will still invalidate the translation, but we can work around that by manually re-entering the translations with the appropriate capitalization in Transifex.